src/cards/types.d.ts

```diff
@@ -49,3 +49,5 @@ type WakaTimeOptions = CommonOptions & {
   layout: "compact" | "normal";
   langs_count: number;
 };
+
+declare module "emoji-name-map";
```

Comment:
Maybe we can use a global.d.ts file to add types for modules that don't export types (see https://github.com/rickstaa/github-emoji-picker/blob/main/src/%40types/global.d.ts).

Reply:
In that case you don't need to add the type folder to each subfolder (i.e. [src/types/emoji-name-map.d.ts](https://github.com/anuraghazra/github-readme-stats/pull/2153/files#diff-de7818750ffd9d577618cebc1f56f789acee279645c6693f698919d8025ffc2b)).

Reply:
I just have deleted them.

Reply:
It's a good idea to use the global.d.ts like that